src/pip/_internal/commands/list.py

```diff
@@ -36,7 +42,9 @@ class ListCommand(IndexGroupCommand):
       %prog [options]"""
 
     def __init__(self, *args, **kw):
-        super(ListCommand, self).__init__(*args, **kw)
+        # type: (*Any, **Any) -> None
+        # https://github.com/python/mypy/issues/4335
+        super(ListCommand, self).__init__(*args, **kw)  # type: ignore
```

Comment:
Is this `type: ignore` still necessary with `(Any, Any)` or `(*Any, **Any)`?

Reply:
No it's not needed. Will remove.